blocks/main/why-kotlin/why-kotlin.tsx

```diff
@@ -0,0 +1,148 @@
+    ];
+
+    return (
+        <ThemeProvider theme={'dark'}>
+            <section className={styles.whyKotlin}>
+                <div className={cn('ktl-layout', 'ktl-layout--center')}>
+                    <div className={cn(darkTextCn('rs-h1'), styles.sectionTitle)}>Why Kotlin</div>
+
+                    <div className={styles.codeExamples}>
+                        <div className={styles.navigationBar}>
+                            <div className={styles.mobileMenuButton}>
+                                <NavItem icon={<DownIcon />} iconPosition={'right'} onClick={handleMobileMenuToggle}>
+                                    {codeExamplesList[activeIndex].children}
+                                </NavItem>
+
+                                <SidebarMenu
+                                    before={
+                                        <SidebarMenuHeader>
+                                            <div className={darkTextCn('rs-text-2', { hardness: 'hard' })}>
+                                                Code examples
+                                            </div>
+                                        </SidebarMenuHeader>
+                                    }
+                                    isOpen={mobileMenuVisible}
+                                    items={codeExamplesList}
+                                    activeIndex={activeIndex}
+                                    onClose={() => setMobileMenuVisible(false)}
+                                    onItemClick={(item, i) => handleMobileMenuItemClick(i)}
+                                />
+                            </div>
+
+                            <div className={styles.tabList}>
+                                {codeExamplesList.map((item, index) => (
+                                    <NavItem
+                                        key={index}
+                                        onClick={() => setActiveIndex(index)}
+                                        active={activeIndex === index}
+                                    >
+                                        {item.children}
+                                    </NavItem>
+                                ))}
+                            </div>
+
+                            <div className={styles.controlButtons}>
+                                <div className={styles.openInPlaygoundButton}>
+                                    <NavItem icon={<CodeIcon />}>Open in Playground</NavItem>
+                                </div>
+
+                                <NavItem icon={<PlayIcon />} onClick={handleRunButton}>
+                                    Run
+                                </NavItem>
+                            </div>
+                        </div>
+
+                        <div className="tab-content kotlin-code-examples-section">
+                            <ContentSwitcher index={activeIndex}>
+                                <div className={styles.tab} key={0}>
```

Comment:
Looks strange `key` without array 😈

Reply:
It's a tab component hack, to it knows that it's different content inside the tab 😂. I'll add samples to `codeExamplesList` and do a map here, good catch